watchman/views.py

```diff
@@ -46,6 +57,8 @@ def status(request):
 
     check_list, skip_list = _get_check_params(request)
 
+    _optional_apm_ignore_transaction()
```

Comment:
Not 100% on this, but the "optional" bit here feels a bit weird - thoughts on doing an explicit check outside of the function? Adds a bit of duplication but feels a bit nicer to me. Might also just rename the function to match the setting.

```python
if settings.WATCHMAN_DISABLE_APM:
    _disable_apm()
```